examples/collector/src/Makefile

```diff
@@ -0,0 +1,69 @@
+# http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+#
+# Copyright 2016 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+PLUGINTYPE=collector
+NAME=rando
+TARGET=test-$(PLUGINTYPE)-$(NAME)
+
+SNAPLIBS=/usr/local/lib
+ICCROOT=
+EROOT=../..
+BDIR=$(EROOT)/bin
+
+CC=g++
+IDIRS=-I.
+LDIRS=-L.
+CFLAGS=-std=c++1y
+LIBS=-Wl,--start-group $(SNAPLIBS)/libsnap.a $(SNAPLIBS)/libgrpc.a $(SNAPLIBS)/libgrpc++.a $(SNAPLIBS)/libprotobuf.a -Wl,--end-group -lz -lpthread
```

Comment:
@aarontay I need to add `-lssl -lcrypto` in this line to build plugin (the same for processor and publisher), without changes I see following errors:
```
/usr/local/lib/libgrpc.a(ssl_transport_security.o):(.text+0xd9): undefined reference to `SSL_free'
/usr/local/lib/libgrpc.a(ssl_transport_security.o):(.text+0x10c): undefined reference to `SSL_free'
/usr/local/lib/libgrpc.a(ssl_transport_security.o):(.text+0x153): undefined reference to `SSL_read'
/usr/local/lib/libgrpc.a(ssl_transport_security.o):(.text+0x1b6): undefined reference to `SSL_get_error'
/usr/local/lib/libgrpc.a(ssl_transport_security.o):(.text+0x273): undefined reference to `ERR_error_string_n'
/usr/local/lib/libgrpc.a(ssl_transport_security.o):(.text+0x29a): undefined reference to `ERR_get_error'
/usr/local/lib/libgrpc.a(ssl_transport_security.o):(.text+0x342): undefined reference to `BIO_write'
/usr/local/lib/libgrpc.a(ssl_transport_security.o):(.text+0x441): undefined reference to `X509_get_ext_d2i'
/usr/local/lib/libgrpc.a(ssl_transport_security.o):(.text+0x455): undefined reference to `sk_num'
/usr/local/lib/libgrpc.a(ssl_transport_security.o):(.text+0x49a): undefined reference to `X509_get_subject_name'
/usr/local/lib/libgrpc.a(ssl_transport_security.o):(.text+0x4ba): undefined reference to `X509_NAME_get_index_by_NID'
/usr/local/lib/libgrpc.a(ssl_transport_security.o):(.text+0x4d2): undefined reference to `X509_NAME_get_entry'
/usr/local/lib/libgrpc.a(ssl_transport_security.o):(.text+0x4e3): undefined reference to `X509_NAME_ENTRY_get_data'
/usr/local/lib/libgrpc.a(ssl_transport_security.o):(.text+0x4f9): undefined reference to `ASN1_STRING_to_UTF8'
/usr/local/lib/libgrpc.a(ssl_transport_security.o):(.text+0x53b): undefined reference to `CRYPTO_free'
/usr/local/lib/libgrpc.a(ssl_transport_security.o):(.text+0x565): undefined reference to `BIO_s_mem'
/usr/local/lib/libgrpc.a(ssl_transport_security.o):(.text+0x56d): undefined reference to `BIO_new'
...
```